docs/4.0.0/rules/index.md

```diff
@@ -0,0 +1,117 @@
+---
+title: List of available rules
+layout: doc
+---
+
+# Rules
+
+Rules in ESLint are grouped by category to help you understand their purpose.
+
+No rules are enabled by default. The `"extends": "eslint:recommended"` property in a [configuration file](../user-guide/configuring#extending-configuration-files) enables rules that report common problems, which have a check mark (recommended) below.
+
+The `--fix` option on the [command line](../user-guide/command-line-interface#fix) automatically fixes problems (currently mostly whitespace) reported by rules which have a wrench (fixable) below.
```

Comment:
Hard to figure out what you were pointing to, but I think it should be all fine, since it's relative paths. All of the documentation was copied into 4.0 folder. So `user-guide` folder is there too.